models/user.js

```diff
@@ -1,32 +1,8 @@
-// Requiring bcrypt for password hashing. Using the bcrypt-nodejs version as the regular bcrypt module
-// sometimes causes errors on Windows machines
-const bcrypt = require("bcrypt-nodejs");
-// Creating our User model
+'use strict';
```

Comment:
What is this?

Reply:
ES6 strict mode that I'm trying out for our project. The benefits are it prevents throw errors and it will make the code less mutable